src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -755,11 +755,13 @@ def create_arm_client_factory(credentials):
                 return arm_client_factory(credentials)
             from azure.cli.core.profiles._shared import get_client_class
             from azure.cli.core.profiles import ResourceType, get_api_version
-            from azure.cli.core._debug import change_ssl_cert_verification
+            from azure.cli.core.commands.client_factory import configure_common_settings
             client_type = get_client_class(ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS)
             api_version = get_api_version(cli_ctx, ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS)
-            return change_ssl_cert_verification(client_type(credentials, api_version=api_version,
```

Comment:
this func will disable ssl cert verify per env variable, replace it completely will lose the ability, is it possible to append user agent separately instead of configure_ocmmon_settings?

Reply:
`change_ssl_cert_verification` is already included in `configure_common_settings`, so this will only extend the logic, instead of replacing it. 
https://github.com/Azure/azure-cli/blob/d1613b0d235327c766e448f89362526e2c15fb90/src/azure-cli-core/azure/cli/core/commands/client_factory.py#L79-L80